### PR TITLE
register NANPA prefix resource in ruby SDK

### LINKED ISSUE #28: register NANPA prefix resource in ruby SDK that will be eveilable in DIDWW API3 (v3.3)
usage examples:

```
require 'didww'

c = DIDWW::Client.configure do |config|
  config.api_key  = 'api_key'
end

united_states = c.countries.where(iso: 'US').first
sc_region = c.regions.where(name: 'South Carolina').first

npa_nxx_prefixes = c.nanpa_prefixes.where('region.id': sc_region.id, 'country.id': united_states.id).all

did_groups = c.did_groups.where('nanpa_prefix.id': npa_nxx_prefixes.first.id).all

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
usage examples:

```
require 'didww'

c = DIDWW::Client.configure do |config|
  config.api_key  = 'api_key'
end

united_states = c.countries.where(iso: 'US').first
sc_region = c.regions.where(name: 'South Carolina').first

npa_nxx_prefixes = c.nanpa_prefixes.where('region.id': sc_region.id, 'country.id': united_states.id).all

did_groups = c.did_groups.where('nanpa_prefix.id': npa_nxx_prefixes.first.id).all
```
closes #28

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- registered new resource named: nanpa_prefixes from [#25](https://github.com/didww/didww-v3-ruby/pull/25)
```
Comment (gigorok) [inline CHANGELOG.md:9]:
```suggestion
- /v3/nanpa_prefixes endpoints being added.
```

File: lib/didww/resource/nanpa_prefix.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      has_one :country, class_name: Country
+      has_one :region, class_name: Region
+
+      property :npa, type: :string
+      # Type: String
+      # Description: NPA codes, or area codes consisting of three digits
+      property :nxx, type: :string
+      # Type: String
+      # Description: Refer to three-digit code
+      # that forms the second part of a 10-digit North American phone number
```
Comment (gigorok) [inline lib/didww/resource/nanpa_prefix.rb:15]: why are comments behind the code?

File: spec/fixtures/nanpa_prefixes/get/include/rspec.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "errors": [
```
Comment (gigorok) [inline spec/fixtures/nanpa_prefixes/get/include/rspec.json:2]: where this fixture is used?

File: lib/didww/resource/nanpa_prefix.rb (hunk 2 above)
Comment (Ivanov-Anton) [reply lib/didww/resource/nanpa_prefix.rb:15]: because this style is used for entire gem

**Review by gigorok — APPROVED** [review]

**Pull request merged**